### PR TITLE
Change menu logo

### PR DESCRIPTION
**Pull request by kristophesankar**
Related to #799

Added `url-loader` from [npm](https://www.npmjs.com/package/url-loader) to be able to load truetype fonts.
Added the hobo.ttf font.
Updated the styles.
Removed old img logo from UI.
Added focallocal text link to UI.

### CODE REVIEW

**Comment by AndyatFocallocal** [comment]
looks good, i'm merging now @kristophesankar . Can you please check the live site in 15mins to see that it's displaying how you expected :)

**Comment by kristophesankar** [comment]
Works as expected in desktop. Seems I forgot to change the color on mobile. Should I update the branch and make a new PR?

**Comment by AndyatFocallocal** [comment]
awesome, well done. yes please, i'll merge your PR tomorrow

File: imports/client/ui/includes/MainMenu/Logo/index.js [hunk 1]
```diff
@@ -5,18 +5,22 @@ import { NavbarBrand } from 'reactstrap'
...
   return (
     <NavbarBrand id='brand-logo' tag='div' onClick={onClick}>
       <NavLink to='/' exact>
-        <img src={'/images/' + imgUrl} />
+        {
+          window.__mapType === 'gatherings'
+            ? <a class="brand-logo brand-text" href="http://focallocal.org">
+                Focallocal
+              </a>
+            : <img src={'/images/' + imgUrl} />
```
Comment (artysidorenko) [inline imports/client/ui/includes/MainMenu/Logo/index.js]: I think as you changed imgUrl to btmImgUrl at the top of the file, this throws an error when mapType is set to btm

**Comment by artysidorenko** [comment]
Hey @kristophesankar I think there's a typo in Logo/index.js which only becomes apparent when the mapType is set to btm, that stops the site from loading. Added a comment in the code that explains (renamed imgUrl to btmImgUrl in one place but I think the old name is still referenced elsewhere in the file).

**Comment by kristophesankar** [comment]
Hi @artysidorenko thanks for letting me know. Will update the name back to what it previously was.

**Pull request merged**